src/ProposalTypesConfigurator.sol

```diff
@@ -164,12 +164,59 @@ contract ProposalTypesConfigurator is IProposalTypesConfigurator {
         if (!_proposalTypesExists[proposalTypeId]) revert InvalidProposalType();
 
         if (!scopeExists[proposalTypeId][txTypeHash]) revert InvalidScope();
-        Scope[] memory validScopes = scopes[proposalTypeId];
+        Scope memory validScope = scopes[proposalTypeId][txTypeHash];
+        return validScope.encodedLimits;
+    }
+
+    function getParameter(bytes calldata limit, uint256 startIdx, uint256 endIdx)
+        public
+        pure
+        returns (bytes memory parameter)
+    {
+        return limit[startIdx:endIdx + 1];
+    }
+
+    function validateProposedTx(bytes calldata proposedTx, uint8 proposalTypeId, bytes32 txTypeHash)
+        public
+        view
+        returns (bool valid)
```

Comment:
is there a case where we would want to continue if the proposed tx is not valid (i.e. returns false)? we don't have any logic that could return false so i think we can remove this return and the func will revert if there's something invalid

Reply:
if someone wants to skip any reverts, they can use a low level call or try/catch

Reply:
I was thinking of it having revert on the proposal creation step, and have this being used in the front end at first when the user is building a scope.

Reply:
Figured just a boolean value would be cleaner to handle.